CMakeLists.txt

```diff
@@ -426,12 +517,14 @@ if(BOOTSTRAP)
     return()
 endif()
 
+message("-------- BOOTSTRAP BEGIN --------")
```

Comment:
hmm thanks for the reminder. I don't think this process is needed any longer.

Reply:
It took me a while to figure out why things were called twice with different parameterization, that's why I put a separator there.